### PR TITLE
Clarified where to download vic engine, and not to download UI files.

### PR DESCRIPTION
**Pull request by stuclem**
Fixes https://github.com/vmware/vic-product/issues/311.

@andrewtchin can you please take a look? Thanks!

### CODE REVIEW

**Comment by stuclem** [comment]
BTW, @andrewtchin this is an update for the 1.1.x docs, not 1.2.

File: docs/user_doc/vic_vsphere_admin/deploy_vic_appliance.md [hunk 1]
```diff
@@ -57,9 +58,13 @@ You install vSphere Integrated Containers by deploying a virtual appliance. The
...
+- Download the vSphere Integrated Containers Engine binaries bundle, `vic_1.1.x.tar.gz`, from the file server. Unpack the bundle and start deploying virtual container hosts (VCHs). For information about deploying VCHs, see [Using vic-machine to Deploy VCHs](deploy_vch.md). 
+- Install the vSphere Client plug-ins for vSphere Integrated Containers.  
+
+  **NOTE**: Do not download the client plug-in files, `com.vmware.vic-v1.1.x.zip` and `com.vmware.vic.ui-v1.1.x.zip`, directly from the file server. You install the vSphere Integrated Containers plug-ins by running a script that pulls these files from the file server. For information about installing the plug-ins, see [Installing the vSphere Client Plug-ins](install_vic_plugin.md).
+
+- Log in to vSphere Integrated Containers Registry at https://<i>vic_appliance_address</i>:443. If the vSphere Integrated Containers appliance uses a different port for vSphere Integrated Containers Registry, replace 443 with the appropriate port. For information about how to use vSphere Integrated Containers Registry, see [Managing Images, Projects, and Users with vSphere Integrated Containers Registry](../vic_dev_ops/using_registry.md).
```
Comment (sgairo) [inline docs/user_doc/vic_vsphere_admin/deploy_vic_appliance.md]: Just curious, is vSphere Integrated Containers Registry referencing Harbor?  Or something separate?

Comment (stuclem) [inline docs/user_doc/vic_vsphere_admin/deploy_vic_appliance.md]: @sgairo yes, that's the official name for Harbor :)

**Comment by stuclem** [comment]
Thanks @andrewtchin and @sgairo !

**Pull request merged**